src/main/resources/edu/cornell/kfs/module/cam/cu-spring-cam.xml

```diff
@@ -45,7 +45,10 @@
  	     
 	<bean id="assetDepreciationService"  parent="assetDepreciationService-parentBean" class="edu.cornell.kfs.module.cam.batch.service.impl.CuAssetDepreciationServiceImpl" />
  	<bean id="assetService"  parent="assetService-parentBean"
- 	     class="edu.cornell.kfs.module.cam.document.service.impl.CuAssetServiceImpl" />
+ 	     class="edu.cornell.kfs.module.cam.document.service.impl.CuAssetServiceImpl" >
+ 	     <property name="businessObjectService" ref="businessObjectService" />
+ 	     <property name="parameterService" ref="parameterService" />
```

Comment:
The parent bean is already injecting parameterService and businessObjectService, so I don't think it's necessary to override the injection here.